packages/backend/src/managers/catalogManager.ts

```diff
@@ -223,6 +225,7 @@ export class CatalogManager extends Publisher<ApplicationCatalog> implements Dis
           },
           memory: statFile.size,
           backend: local.backend ?? InferenceType.NONE,
+          sha256: sha256,
```

Comment:
question: are we not expecting that sha256 is the sha of the model I suppose, not the sha of the path ?